build-logic/src/main/kotlin/Publishing.kt

```diff
@@ -80,25 +83,64 @@ fun Project.configureDokka() {
   dokkatoo.dokkatooSourceSets.configureEach {
     includes.from("README.md")
   }
-  if (this == rootProject) {
-    dependencies.add(
-        "dokkatooPluginHtml",
-        dokkatoo.versions.jetbrainsDokka.map { dokkaVersion ->
-          "org.jetbrains.dokka:all-modules-page-plugin:$dokkaVersion"
-        }
-    )
-    dependencies.add(
-        "dokkatooPluginHtml",
-        dokkatoo.versions.jetbrainsDokka.map { dokkaVersion ->
-          "org.jetbrains.dokka:versioning-plugin:$dokkaVersion"
-        }
-    )
-    dokkatoo.pluginsConfiguration.getByName("versioning") {
-      this as DokkaVersioningPluginParameters
-      olderVersionsDir.set(file("build/kdoc-versions"))
+
+  return dokkatoo
+}
+
+fun Project.configureDokka() {
+  configureDokkaCommon()
+  val project = this
+  val kdocProject = project(":apollo-kdoc")
+  kdocProject.configurations.all {
+    if (name == "dokkatoo" && project.name == "apollo-annotations") {
+      dependencies.add(kdocProject.dependencies.project(mapOf("path" to project.path)))
```

Comment:
Not sure how "project isolation compatible" this is. Problem for another day....